config/routes.rb

```diff
@@ -263,9 +263,9 @@
     end
     scope '/verify', module: 'idv', as: 'idv' do
       get '/come_back_later' => 'come_back_later#show'
-      get '/confirmations' => 'confirmations#show'
-      post '/confirmations' => 'confirmations#update'
```

Comment:
We'll want to keep the old routes for backwards compatibility during the deploy

Reply:
Yes! see a4716c2ac

Reply:
@zachmargolis Can we revisit to remove these routes now?

(Noticed while auditing similar code)

Reply:
Yes, good catch!